lib/ex_doc/formatter/html/autolink.ex

```diff
@@ -325,7 +325,7 @@ defmodule ExDoc.Formatter.HTML.Autolink do
   #
   # `language` can be: `:elixir`, `:erlang` or `:markdown`.
   #
-  # `kind` is either `:function` or `:module`.
+  # `kind` is either `:function`, `:module`, or `:mix_task`.
```

Comment:
I'm OK with :mix_task, but I'm wondering. Is there any other kind of task?
Should it be called just `task`?
is there a equivalent of a Mix Task in Erlang?

Reply:
beacuse I'm seeing we are calling `{:mix_task, :elixir, :normal}`, it is a bit redundan i guess.
`{:mix_task, :elixir, :normal}`, and in the future we could call `{:task, :erlang, :normal}`

My only concern is that there are other tools other than Mix and Rebar for creating tasks that we may support

Reply:
:+1: for `mix_task` because that's also what we call it in the sidebar.